blocks/career-carousel/career-carousel.css

```diff
@@ -0,0 +1,148 @@
+.career-carousel {
+  --card-width: 16rem;
+}
+
+.career-slider {
+  display: flex;
+  flex-direction: column;
+  margin-left: -1rem;
+  width: calc((var(--card-width) + 3rem) * 4);
+  overflow: hidden;
+}
+
+.career-slides {
+  display: flex;
+  overflow-x: scroll;
+  scroll-behavior: smooth;
+
+  /* We want to hide the scroll bar */
+  -ms-overflow-style: none; /* for Internet Explorer, Edge */
+  scrollbar-width: none; /* for Firefox */
+}
+
+/* Hide the scroll bar */
+.career-slides::-webkit-scrollbar {
+  /* for Chrome, Safari and Opera */
+  display: none;
+}
+
+.career-carousel .career-card {
+  flex-shrink: 0;
+  box-shadow: #3c465040 0 2px 12px 0;
+  margin: 0.5rem;
+  padding: 1rem;
+  width: var(--card-width);
+  transition: all 0.15s ease-in-out;
+}
+
+.career-carousel .career-card:hover {
+  box-shadow: #3c465066 0 4px 16px 0;
+}
+
+.career-carousel .career-card a {
+  display: inline-flex;
+  flex-direction: column;
+  text-decoration: none;
+}
+
+.career-carousel .career-card picture {
+  align-self: center;
+  width: 14.375rem;
+  margin: 0 0 1.375rem;
+}
+
+.career-carousel .career-card picture img {
+  height: 16rem;
+  width: 14.375rem;
+}
+
+.career-carousel .career-card blockquote {
+  color: var(--primary);
+  font-size: 19px;
```

Comment:
i think it would be good to use standard font-size here, you can map it to 18px
--body-font-size-l